rust/src/lib/ifaces/ovs.rs

```diff
@@ -570,6 +585,16 @@ impl OvsBridgeBondConfig {
         Self::default()
     }
 
+    pub(crate) fn post_deserialize_cleanup(&mut self) {
+        if self.slaves.as_ref().is_some() {
+            log::warn!(
+                "The `slaves` is deprecated, please replace with `ports`."
+            );
```

Comment:
I am using the format command, I did not see the warning with `-v` option. 

I also noticed that this patch actually fixed another bug: 
``` // the binary build on base branch and format with the example provided in the commit message 
[root@fcvm rust]# ./target/debug/nmstatectl  -vv format test.yml  
dns-resolver: {}
route-rules: {}
routes: {}
interfaces:
- name: br0
  type: ovs-bridge
  state: up
  bridge:
    port:
    - name: ovs0
    - name: bond1
      link-aggregation:
        mode: balance-slb
```

 ```//the binary build on current patch and format with the example provided in the commit message
 [root@fcvm rust]# ./target/debug/nmstatectl  -vv format test.yml  
dns-resolver: {}
route-rules: {}
routes: {}
interfaces:
- name: br0
  type: ovs-bridge
  state: up
  bridge:
    port:
    - name: ovs0
    - name: bond1
      link-aggregation:
        mode: balance-slb
        port:
        - name: eth2
        - name: eth1
```